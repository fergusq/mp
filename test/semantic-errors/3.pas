program p;
begin procedure f();
begin writeln(1)end;
end.
