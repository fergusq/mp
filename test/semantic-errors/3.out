Warning: Program is not Mini-Pascal 2018 compliant: functions and procedures should be declared at the top level only
